lms/envs/common.py

```diff
@@ -2662,6 +2662,7 @@
 
     # Enterprise App (http://github.com/edx/edx-enterprise)
     'enterprise',
+    'django_object_actions',
```

Comment:
It doesn't really hurt to have it in more than one PR.  I can remove it if you prefer, but we might get this PR merged faster than the other one.